### PR TITLE
feat(generator-pie-component): DSW-1054 add rtl  support to the gener…

### PR DESCRIPTION
**Pull request by raoufswe**
[Added] - RTL option to the pie web component generator

## Author Checklist (complete before requesting a review)
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] If it is a `PIE Docs` change, I have reviewed the Docs site preview
- [ ] If it is a component change, I have reviewed the Storybook preview
- [ ] If there are visual test updates, I have reviewed them properly before approving

### Reviewer 1
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### Reviewer 2
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### CODE REVIEW

**Review by fernandofranca — APPROVED** [review]

File: packages/tools/generator-pie-component/src/app/templates/pie-placeholder.__stories__.ts [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
 
 // TODO: Remove this const when other exports from Pie<%= componentName %> are used on Stories, 
 // otherwise tree-shaking will get rid of the web component definition
 const keptReference = Pie<%= componentName %>;
 
 type <%= componentName %>StoryMeta = StoryMeta<<%= componentName %>Props>;
 
-const defaultArgs: <%= componentName %>Props = {};
+const defaultArgs: <%= componentName %>Props = {
+    <% if (needsRTL) { %>dir: 'rtl',<% } %>
```
Comment (kevinrodrigues) [inline packages/tools/generator-pie-component/src/app/templates/pie-placeholder.__stories__.ts:14]: I think this one should default to LTR e.g `dir: 'ltr'`.

Comment (raoufswe) [reply packages/tools/generator-pie-component/src/app/templates/pie-placeholder.__stories__.ts:14]: updated!

**Review by kevinrodrigues — APPROVED** [review]

File: packages/components/pie-webc-testing/src/playwright/fixtures.ts [hunk 2]
```diff
@@ -13,8 +13,8 @@ interface ExtendedTestContext {
 export const test = baseTest.extend<ExtendedTestContext>({
     makeAxeBuilder: [async ({ page }: { page: Page }, use: (builder: () => AxeBuilder) => Promise<void>) => {
         const makeAxeBuilder = () => new AxeBuilder({ page })
-        .withTags(['wcag21a', 'wcag21aa', 'wcag143', 'cat.color', 'cat.aria'])
-        .disableRules(['color-contrast-enhanced']);
+          .withTags(['wcag21a', 'wcag21aa', 'wcag143', 'cat.color', 'cat.aria'])
```
Comment (fernandofranca) [inline packages/components/pie-webc-testing/src/playwright/fixtures.ts:16]: What was missing to make the indentation work?

Comment (raoufswe) [reply packages/components/pie-webc-testing/src/playwright/fixtures.ts:16]: i did it manually :D

**Review by fernandofranca — APPROVED** [review]

**Pull request merged**